src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static String getRandomColor() {
+        int random = new Random().nextInt(3);
```

Comment:
please pay attention to the checklist: "All magic numbers in your code should be constants.
"

Reply:
I did it.